xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -2003,8 +2004,12 @@ struct AppModel: ModelProtocol {
                 )
                 return Action.succeedIndexPeer(peer)
             } catch {
+                // We may be unable to resolve a peer, but attempt to grab the DID
+                let sphere = try? await environment.noosphere.traverse(petname: petname)
+                let identity = try? await sphere?.identity()
```

Comment:
@bfollington This would result in a double-traversal in case of failure. This seems problem-prone, especially if the failure was with traversal (two network requests?).

Rather than adding the missing did to this action, I would consider adding information to the logging within `indexPeer`.

For example, if we moved this log below the call to `sphere.identity()`, we should be able to log the identity along with the petname. https://github.com/subconsciousnetwork/subconscious/blob/b4b843519e0356304187cad76f3f5717283800f5/xcode/Subconscious/Shared/Services/DataService.swift#L123-L129

IIRC if we have successfully traversed, then we should have the did available. Is my assumption correct?

While in there, I would consider changing all the logging in indexPeer from `logger.debug` to `logger.log`.